classification/models/clinical_context_models.py

```diff
@@ -28,7 +29,7 @@
 from snpdb.models import Lab
 from snpdb.models.models_variant import Allele
 
-clinical_context_signal = django.dispatch.Signal()  # args: "clinical_context", "status", "is_significance_change", "cause"
+clinical_context_signal = django.dispatch.Signal()  # args: "clinical_context", "status", "is_significance_change", "ClinicalContextChangeData"
```

Comment:
the comment here are the parameter names, not the class of the data, should be `clinical_context_change_data`
or best of both worlds `clinical_context_change_data:ClinicalContextChangeData`